packages/hdwallet-provider/index.d.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  JSONRPCRequestPayload,
+  JSONRPCResponsePayload
+} from 'ethereum-protocol';
```

Comment:
Looks like this is missing from the deps. (Wait, where does this sort of thing go... `dependencies` or `devDependencies`?)

Reply:
`web3-provider-engine` use an `ethereum-protocol`. Better add this to `dependencies`?

Reply:
I also noticed that you did not add `debug` to `dependcies`

Reply:
Yep, best to add all dependencies directly! Good catch on `debug`, that should be there too!